modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,8 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {ProductShortName} fixed issues
 
+* link:https://issues.redhat.com/browse/MAISTRA-2378[MAISTRA-2378] When the cluster is configured to use OpenShiftSDN with `ovs-multitenant` and the mesh contains a large number of namespaces (200+), the {product-title} networking plugin is unable to configure the namespaces as quickly as we expected. {ProductShortName} times out causing namespaces to be continuously dropped from the service mesh and then reenlisted.
```

Comment:
I'm not sure about the "as quickly as we expected" part but can't think of anything better to include at this time.  Otherwise lgtm